metadoc-js/src/main/scala/monaco/Monaco.scala

```diff
@@ -2624,3 +2671,27 @@ package worker {
 object Monaco extends js.Object {
   type MarkedString = String | js.Any
 }
+
+package services {
+
```

Comment:
I suggest we put the service definitions in a separate`MonacoServices.scala` file and document rigorously anything that would help a future upgrade or Monaco.